test/IDE/structure.swift

```diff
@@ -42,11 +42,18 @@ struct MyStruct {
   static func cfoo()
 }
 
+enum MyError: Error {
+  case x
+}
+
 // CHECK: <protocol>protocol <name>MyProt</name> {
 // CHECK:   <ifunc>func <name>foo()</name></ifunc>
 // CHECK:   <ifunc>func <name>foo2()</name> throws</ifunc>
 // CHECK:   <ifunc>func <name>foo3()</name> throws -> <type>Int</type></ifunc>
-// CHECK:   <ifunc>func <name>foo4<<generic-param><name>T</name></generic-param>>()</name> where T: MyProt</ifunc>
+
+// FIXME: The end of the source range needs to be advanced past the ')' here!
+// CHECK:   <ifunc>func <name>foo4()</name> throws(MyError</ifunc>)
```

Comment:
Note that this is not really a new bug. Before this change it was `throws</ifunc>(MyError)`, and now the end has moved but it's still incorrect